### PR TITLE
add machine-api-provider-nutanix repo

### PR DESCRIPTION
**Pull request by elmiko**
This change adds the new repository for the nutanix machine api
provider, it was created by running `make new-repo`.

### CODE REVIEW

File: ci-operator/config/openshift/machine-api-provider-nutanix/openshift-machine-api-provider-nutanix-main.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      memory: 200Mi
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: unit
+  commands: make test
+  container:
+    from: src
```
Comment (JoelSpeed) [inline ci-operator/config/openshift/machine-api-provider-nutanix/openshift-machine-api-provider-nutanix-main.yaml:39]: Is there a makefile in the repo yet? Will this actually work?

**Review by JoelSpeed — COMMENTED** [review]
It would be good to add the `_prowconfig` for this repo too so that we get automated merging via tide

File: ci-operator/config/openshift/machine-api-provider-nutanix/openshift-machine-api-provider-nutanix-main.yaml (hunk 1 above)
Comment (JoelSpeed) [reply ci-operator/config/openshift/machine-api-provider-nutanix/openshift-machine-api-provider-nutanix-main.yaml:39]: Not a blocker assuming there is definitely going to be a `make test` target

**Comment by elmiko** [comment]
just a note here, i am working to add the nutanix code into the openshift repo. they do have a makefile with most of the basic targets there.

**Comment by elmiko** [comment]
https://github.com/openshift/machine-api-provider-nutanix/pull/2 adds the makefile to the target repo

**Comment by hongkailiu** [comment]
/lgtm

give a chance to get the Makefile in first.
/hold

**Comment by elmiko** [comment]
talked with nutanix, they are going to disable the tests which make calls to the external endpoints. we agreed that these should be in the e2e's but not necessarily in the unit tests. will update when we have a patch ready

**Pull request merged**